### ISSUE TITLE
[ADE Support]:Clinical Reminders, Health Questionnaire Pilot - Notable COTS Accessibility Assessment Windows for desktop (JAWS)

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#113005]
> [!NOTE]
> **QUICK SUMMARY:** _Child ticket of #112798  to track accessibility assessment of COTS on Windows for desktop (JAWS)._

## Description of the Work
The team is looking for an accessibility/useability audit of the software using AT. Please add any ally violations/usability issues you find in the main ticket, with a rating of how severe the issue is. No need to provide recommended fix.

## Supporting Artifacts 
Below are the links to the Notable prototype along with log in credentials. They are active for 72 hours after the vendor send them to me. These expire on 6/28.

https://app.notablehealth.com/previsit/app/#/login/LQVZOSGGZUH9
- DOB: 03/09/1945
- Last Name: FIVE

Login is restricted to 1 user, and once experience is completed (by getting the final page), you can no longer renter the same experience. However, you can re-enter an experience to complete remaining questions.

## ADE Tasks
> [!TIP]
> Customize ADE Tasks as needed. 
- [ ] Collaborate `task`
- [ ] ADE Support `task`
- [ ] Document `task`
- [x] Review `task`

## Acceptance Criteria
> [!TIP]
> Customize ADE Acceptance Criteria as needed. 
- [ ] Acceptance Criteria `Example 1`
- [ ] Acceptance Criteria `Example 2`
- [ ] Acceptance Criteria `Example 3`

### Ticket Updates
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`

**Comment by coforma-jamie** [comment]
(in progress)

All pages

- No H1 on page
- No skip link
- Focus isn't managed, nothing is announced upon entering each page
- Tab order doesn't match visual order. It should be Back button, Logout link, question 1, question 2 (if applicable), Continue button. But it's actually question 1, Back button, Logout link, question 2 (if applicable), Continue button
- Radio buttons don't allow for standard keyboard controls (must use TAB to navigate between options rather than UP / DOWN ARROW)
- Textarea visually shows the character count, but doesn't announce it to JAWS unless user is in Browse Mode and selects it
- <title> is always "Notable," doesn't change by page or even indicate what type of survey is being taken
- Loading indicator ("Loading...") doesn't announce to assistive technology

Login screen

- No H1 on the page
- Month, day, and year fields don't have labels. JAWS announces their placeholder text -- "MM," "DD," "YYYY" -- but not all screen readers will do that
- Patient last name field doesn't have a label. JAWS announces the placeholder text, but not all screen readers will do that, and the text disappears upon entry
- Month and day fields -- the first time accessing them, I can enter a single number ("5") and the full number ("05") is autofilled. If I SHIFT + TAB back into one of these fields and enter a different single number ("7"), the entered input doesn't change and focus automatically moves to the next field (from DD to YYYY, for example). This really isn't usable for keyboard users.
- When I select Terms of Use link, JAWS doesn't announce anything. Focus isn't managed to the Terms of Use page.
- Terms of Use link is announced as a "same page link," but when I open the Terms of Use and select the Back button, the login page reloads -- I lose anything I've entered into the login fields prior to accessing Terms of Use.

Please answer a few questions to help the VA better understand your medical needs

- Start buttons have duplicate accessible names. They're all "Start" (or "Edit" after finishing a section). JAWS user tabbing through the page doesn't know what each "Start" button is referencing.


How do you think of your sexual orientation?

- User can select all checkboxes, even when that doesn't make sense -- "Choose not to answer" supersedes "Bisexual," for example
- Not an accessibility note -- This is just a strange way to phrase the question. A person doesn't "think of" their orientation, they "are" that orientation, the same way they are a race or ethnicity...I'd change this to a simple "What is your sexual orientation?"

Toxic Exposure Screening entry page

- There's a list of exposures ("Agent Orange," "Radiation," etc) that look like a list but aren't a semantic list (no <li>)
- Unlinked email address
